### PR TITLE
Convert test code to kotlin

### PR DESCRIPTION
**Pull request by utzcoz**
1. Use Android Studio tool to convert test code to kotlin.
2. Fix null checking error, just adding !! and remove some ?.
3. Fix incompatibility of type between Int and Long in assert.

Test: `./gradlew test`

Close https://github.com/farmerbb/Taskbar/issues/177

### CODE REVIEW

**Comment by utzcoz** [comment]
@farmerbb could you help to review this PR? Thanks.

**Comment by utzcoz** [comment]
The generated kotlin project by Android Studio uses `test/java` for kotlin test code, so I keep it.

**Review by farmerbb — CHANGES_REQUESTED** [review]
Could you please change the variables in each test from `private var` to `private lateinit var`?  That way we don't have to worry about overriding the null safety with `!!` when accessing these variables.  I personally prefer to avoid using `!!` wherever possible, only using it as a very last resort.

File: dependencies.gradle [hunk 1]
```diff
@@ -17,5 +17,7 @@ allprojects {
         MATERIAL_VERSION = "1.2.1"
         POWERMOCK_VERSION = "2.0.9"
         AGP_VERSION = "4.1.2"
+
+        KOTILN_VERSION = "1.3.72"
```
Comment (farmerbb) [inline dependencies.gradle:21]: Small typo in `KOTILN_VERSION` - should be `KOTLIN_VERSION`

Comment (utzcoz) [reply dependencies.gradle]: Done.

**Comment by utzcoz** [comment]
> Could you please change the variables in each test from `private var` to `private lateinit var`? That way we don't have to worry about overriding the null safety with `!!` when accessing these variables. I personally prefer to avoid using `!!` wherever possible, only using it as a very last resort.

The new commit processes it.

**Comment by utzcoz** [comment]
The new commit also rebases on `master`.

**Comment by farmerbb** [comment]
Thanks for doing that!

Do you happen to know how we can add Kotlin support to the main codebase while still being able to compile on Android-x86?  I think it involves converting our `Android.mk` file to `Android.bp`, but I'm not sure how to go about doing that and I'm not setup to compile Android-x86.

**Comment by utzcoz** [comment]
> Thanks for doing that!
> 
> Do you happen to know how we can add Kotlin support to the main codebase while still being able to compile on Android-x86? I think it involves converting our `Android.mk` file to `Android.bp`, but I'm not sure how to go about doing that and I'm not setup to compile Android-x86.

Yes, The `Android.bp` can do it. From AOSP 10, the `SystemUI` uses kotlin to write some logic, and its [`Android.bp`](https://cs.android.com/android/platform/superproject/+/master:frameworks/base/packages/SystemUI/Android.bp) can be an example. If we start to use `Android.bp` for kotlin, we will drop the support for `Android-x86` pie for its later version. I know jon has worked with you to release `Taskbar` prebuilt apk for `BlissOS`, maybe it is the perfect solution for `Android-x86`'s `Taskbar` release, and we can change all source code to kotlin after that.

**Pull request merged**